app/script/cryptography/CryptographyRepository.js

```diff
@@ -50,15 +50,49 @@ z.cryptography.CryptographyRepository = class CryptographyRepository {
   }
 
   /**
+   * Initializes the repository by loading an existing cryptobox.
```

Comment:
`cryptobox` should be capitalized: https://github.com/wireapp/cryptobox#cryptobox